services/api.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const myapi = axios.create({
+  baseURL: "https://nc-news-bpw3.onrender.com",
```

Comment:
all your endpoints probably start with /api? so you could add that to the baseURL